Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/InteractableColorTheme.cs

```diff
@@ -323,6 +353,23 @@ protected bool TrySetRendererColor(Color colour, ThemeStateProperty property, in
             return true;
         }
 
+        /// <summary>
+        /// Try to set color on Image
+        /// If false, no Image was found.
+        /// </summary>
+        protected bool TrySetUnityUiImageColor(Color colour, ThemeStateProperty property, int index, float percentage)
```

Comment:
>colour [](start = 53, length = 6)

British spelling again here.  And actually I see that the file is already inconsistent.  Maybe while you're here, change all of the vars/params named "colour" to "color" for consistency.